functions/src/Twilio/email.ts

```diff
@@ -0,0 +1,36 @@
+import * as sgMail from "@sendgrid/mail";
+import * as functions from "firebase-functions";
```

Comment:
Remove unused imports.

Reply:
It is not unused, I need that for the environment config